### PR TITLE
Autoscaling disk size in Forge

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Edits existing resource allocation content, and adds new autoscaling content.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes https://github.com/timescale/docs/issues/153

### CODE REVIEW

**Comment by Loquacity** [comment]
@tedsczelecki thanks! I was clearly having an asciidoctor moment 😂

**Comment by Loquacity** [comment]
> Link to build for this PR: http://docs-dev.timescale.com/docs-forge-autoscaling-lana

image

File: timescale-forge/scaling-a-service.md [hunk 1]
```diff
@@ -1,52 +1,55 @@
...
+    allocation, your service will go down briefly while the changes are applied.
+    <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tsc-resources-configure.png" alt="Configure resource allocations"/>
+
+## Configure autoscaling for disk size
+Disk size autoscaling is enabled by default on most services. When you consume
+95% or more of your existing disk space, disk size is automatically increased to
+the next size available, up to a configurable limit.
+
+Autoscaling can change the disk size once every six hours, and can only increase
+the size, not decrease it. You can have a disk up to 10{nbsp}TB in size.
```
Comment (lukas-bernert) [inline timescale-forge/scaling-a-service.md:41]:
```suggestion
the size, not decrease it. You can have a disk up to 10&nbsp;TB in size.
```

File: timescale-forge/scaling-a-service.md [hunk 2]
```diff
@@ -1,52 +1,55 @@
...
+
+### Procedure: Configuring autoscaling for disk size
+1.  In the Timescale Forge console, from the `Services` list, click the name of
+    the service you want to modify.
+1.  In the `Service details` page, navigate to the `Operations` tab, and click
```
Comment (lukas-bernert) [inline timescale-forge/scaling-a-service.md:46]: Nit but we usually call this page "Service overview". Also the URL path is named "/dashboard/services/service-id/overview"
Maybe best to stay consistent

File: timescale-forge/scaling-a-service.md [hunk 3]
```diff
@@ -1,52 +1,55 @@
...
+95% or more of your existing disk space, disk size is automatically increased to
+the next size available, up to a configurable limit.
+
+Autoscaling can change the disk size once every six hours, and can only increase
+the size, not decrease it. You can have a disk up to 10&nbsp;TB in size.
+
+### Procedure: Configuring autoscaling for disk size
+1.  In the Timescale Forge console, from the `Services` list, click the name of
+    the service you want to modify.
+1.  In the `Service details` page, navigate to the `Operations` tab, and click
```
Comment (Loquacity) [inline timescale-forge/scaling-a-service.md:46]:
```suggestion
1.  In the `Service overview` page, navigate to the `Operations` tab, and click
```

**Review by lukas-bernert — APPROVED** [review]

**Comment by Loquacity** [comment]
@lukas-bernert Happy to merge this one?

**Comment by lukas-bernert** [comment]
@Loquacity autoscaling is still being implemented. We expect to launch it next week. Once released we can put the docs live.

**Comment by clarkbw** [comment]
docs are live here: https://docs.timescale.com/timescale-forge/latest/scaling-a-service/#procedure-changing-resource-allocations-manually

**Pull request merged**